### PR TITLE
Upgrade asciidoc to 10.2.0

### PR DESCRIPTION
**Pull request by suresh-thelkar**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Upgrade asciidoc to 10.2.0

###### Change Log  

- NA

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- Uniform Buddy Build results are [here](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=484390)

### CODE REVIEW

File: SPECS/asciidoc/asciidoc.spec [hunk 1]
```diff
@@ -47,11 +55,21 @@ python3 tests/testasciidoc.py run
...
 %license COPYRIGHT
+%doc BUGS.adoc CHANGELOG.adoc README.md
 %{_bindir}/*
-%{_sysconfdir}/*
 %{_mandir}/*
+%{python3_sitelib}/asciidoc/
+%{_pkgdocdir}/doc
+%dir %{python3_sitelib}/asciidoc/resources/filters/latex
+%exclude %{python3_sitelib}/asciidoc/resources/filters/music
+%exclude %{python3_sitelib}/asciidoc-10.2.0-py3.9.egg-info
```
Comment (ameyapu) [inline SPECS/asciidoc/asciidoc.spec:65]: Could you brief on these exclusions?

File: SPECS/asciidoc/asciidoc.spec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Summary:        AsciiDoc is a human readable text document format
 Name:           asciidoc
-Version:        9.1.0
+Version:        10.2.0
 Release:        1%{?dist}
 License:        GPLv2
```
Comment (ameyapu) [inline SPECS/asciidoc/asciidoc.spec:5]: https://github.com/asciidoctor/asciidoc-docs/blob/main/README.adoc says some different license, not GPLv2.

**Review by ameyapu — APPROVED** [review]
Other changes look good.  Please address the comments and proceed to merging.

File: SPECS/asciidoc/asciidoc.spec (hunk 2 above)
Comment (suresh-thelkar) [reply SPECS/asciidoc/asciidoc.spec:5]: @ameyapu, Thank you for the review. The repo that we are using is given below. 
https://github.com/asciidoc-py/asciidoc-py/blob/main/LICENSE
According to this It seems the license is GPLv2  only.  I think we are good here. Let me know If I am missing anything.

File: SPECS/asciidoc/asciidoc.spec (hunk 1 above)
Comment (suresh-thelkar) [reply SPECS/asciidoc/asciidoc.spec]: I was originally thinking that music filters are not used by anybody as they were not there in the earlier version of rpm and hence excluded them.  
asciidoc-10.2.0-py3.9.egg-info is actually useful for tools like pip and hence keep it. I have added a commit by including both of them now.

**Pull request merged**